tests/load/test_job_client.py

```diff
@@ -951,6 +952,73 @@ def _load_something(_client: SqlJobClientBase, expected_rows: int) -> None:
             )
 
 
+# NOTE: this could be folded into the above tests, but these only run on sql_client destinations for now
+# but we want to test filesystem and vector db here too
```

Comment:
why can't you extend to other destinations with selectors below?

Reply:
at least filesystem should be tested

Reply:
The schema stuff for the filesystem currently is tested in this new function as well as the filesystem specific tests. I'd like to do a bigger PR soon where I harmonize all the tests that can be merged, but that is a bit of work and will also require some changes in the filesystem that are not at all related to this dataset stuff. Or is there some change here that in your view is not tested?